src/databricks/labs/ucx/install.py

```diff
@@ -202,6 +202,18 @@ def configure(self) -> WorkspaceConfig:
             "Inventory Database stored in hive_metastore", default="ucx", valid_regex=r"^\w+$"
         )
 
+        if self.check_installation_conflicts(inventory_database):
+            raise RuntimeWarning(f"Inventory database with name {inventory_database} already exists")
+
+        self.single_user_install = self._prompts.confirm("Do you want to install for a single user?")
```

Comment:
```suggestion
        if self._prompts.confirm("override global installation?")
            self._installation = Installation.current(self._ws, "ucx", assume_user=True)
```

... but make it show this prompt only if global installation not found